File: PYTHON/Practicals/Practical_20.py
# WAP to find sum of natural numbers using for loop as well as while loop.

import os
os.system('cls')

limit = int(input("\n Enter the Limit of Natural Number to ADD : "))

sum = 0
for i in range(1,limit+1):
    sum += i
print("\n Sum of Natural Number using For Loop --> ",sum)

sum = 0
j = 1
while j < limit+1:
    sum += j
    j += 1
print("\n Sum of Natural Number using While Loop --> ",sum)

File: C-Programming/Practicals/Practical_20.c
// Write a program to find sum of all natural numbers using loop (also use while loop, for
// loop and do while loop).

#include <stdio.h>
main()
{
    int i, x, y = 0;
    printf("\n Enter any Number: ");
    scanf("%d", &x);
    for (i = 1; i <= x; ++i)
    {
        y += i;
    }
    printf("\n Sum of the given input= %d", y);
    getch();
}
